AreaSin.java

```diff
@@ -1,13 +1,29 @@
 class AreaSin extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * A program AreaSin.java that calculates the area of a triangle given two side lengths
+  * and a contained angle.
+  * @author: Kayla Jin 
   */
   
   public void run() {
     
-    // start coding here
+    // Declare variables
+    double dblSideA;
+    double dblSideB;
+    double dblAngleC;
+    double dblTriangleArea;
+
+    // Gather user input
+    dblSideA = readDouble("Enter the length of side A: ");
+    dblSideB = readDouble("Enter the length of side B: ");
+    dblAngleC = readDouble("Enter the degree of angle C: ");
+
+    // Calculate the area of the triangle
+    dblTriangleArea = dblSideA * dblSideB * Math.sin(Math.toRadians(dblAngleC)) / 2;
```

Comment:
The solution puts brackets around everything before the division by 2, but it does not make a difference in the calculations due to the order of operations. The solution also divides by 2.0 instead of 2, which does not matter because the result will be a double either way.